Dockerfile

```diff
@@ -1,7 +1,27 @@
 FROM python:3-alpine
+FROM node:carbon-alpine AS build
+
+FROM ubuntu
```

Comment:
In your configuration here, it's going to be the last `FROM` parameter that wins out, unless you are using [multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/). So you are essentially using a vanilla Ubuntu image then installing Python manually on top of it. You can optimize this more by using the default Python image (`python:3`) which is Ubuntu with Python 3 already installed by default. :ok_hand: